VideoLocker/res/values/styles.xml

```diff
@@ -222,6 +222,21 @@
         <item name="android:maxHeight">3dip</item>
     </style>
 
+    <style name="edX.Widget.Switch" parent="Widget.AppCompat.CompoundButton.Switch">
+        <item name="android:switchTextAppearance">@style/default_switch</item>
+        <item name="android:showText">true</item>
+        <item name="android:textOff">@string/toggle_turn_off</item>
+        <item name="android:textOn">@string/toggle_turn_on</item>
+        <item name="android:thumb">@drawable/switch_inner</item>
+        <item name="android:thumbTextPadding">8dp</item>
+        <item name="android:track">@drawable/switch_track</item>
+        <item name="android:layout_width">90dp</item>
```

Comment:
Since we're already defining the minimum width in the `android:switchMinWidth`, we might as well move the actual width there to make it useful, and set the `android:layout_width` to `wrap_content` as well.